sor/src/main/java/com/bazaarvoice/emodb/sor/core/DatabusEventWriterRegistry.java

```diff
@@ -0,0 +1,32 @@
+package com.bazaarvoice.emodb.sor.core;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * This is used to be used to send events to the databus prior to writing to the system of record.
```

Comment:
Minor: This is used ~~to be used~~ to send events...

Reply:
whoops, I'll fix